app/assets/stylesheets/topbar.scss

```diff
@@ -32,18 +50,13 @@
     height: 30px;
   }
 
-  ul.dropdown.menu li {
-    margin-bottom: 3px;
-    height: 25px;
-  }
-
-  li {
-    & a {
-    padding: 0;
-    }
+  .menu li a {
```

Comment:
Merge rule `.menu li a` with rule on line 29<br>Selector should have depth of applicability no greater than 2, but was 4